make/test.mk

```diff
@@ -263,7 +263,7 @@ get-and-publish-operators: clean-e2e-files get-and-publish-host-operator get-and
 #			 The reason is that when the host operator is installed, then the logic creates ToolchainConfig CR which
 #			 defines that the webhook should be deployed from the first member instance (and not from the second one).
 #			 This is important to set before the member operators are installed, otherwise, it can lead to flaky e2e tests.
-get-publish-install-and-register-operators: get-and-publish-host-operator setup-toolchainclusters create-host-resources get-and-publish-member-operator
+get-publish-install-and-register-operators: get-and-publish-host-operator get-and-publish-member-operator setup-toolchainclusters create-host-resources
```

Comment:
this is needed since now we are deploying the toolchaincluster-member SA from the member-operator deployment, thus we need to run `setup-toolchainclusters` only after deploying the member operator deployment. 

In order to invert those make targets I had todo some changes in member operator, see paired PR for those.